packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -112,18 +117,30 @@ export class AtomicSearchBox implements AtomicSearchBoxOptions {
   }
 
   private onClickSuggestion(e: MouseEvent) {
-    const value = (e.target as HTMLButtonElement).value;
-    this.searchBox.selectSuggestion(value);
+    const value = (e.currentTarget as HTMLLIElement).value;
+    this.searchBox.selectSuggestion(
+      this.searchBoxState.suggestions[value].rawValue
+    );
   }
 
   private get submitButton() {
     return (
       <button
         type="button"
         part="submit-button"
+        class={
+          'w-10 bg-transparent border-0 focus:outline-none border-medium-grey border-solid p-0 ' +
```

Comment:
The contast ratio of `medium-grey` (`#bcc3ca`) against white is 1.78:1. To pass [success criterion 1.4.11](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html#user-interface-components), we should probably try to have a contrast ratio of 3:1.

Don't forget to change it for the search button and clear button too.